ydb/core/kqp/ut/query/kqp_query_ut.cpp

```diff
@@ -1689,6 +1689,258 @@ Y_UNIT_TEST_SUITE(KqpQuery) {
             CompareYson(output, R"([[1u;[1];["test1"]];[100u;[100];["test2"]]])");
         }
     }
+
+
+    void RunQuery (const TString& query, auto& session, bool expectOk = true) {
```

Comment:
I think it's better to move the tests to ydb/core/kqp/ut/service/kqp_qs_queries_ut.cpp